### PR TITLE
add log_b256 method to std library

### PR DESCRIPTION
**Pull request by SilentCicero**
- Adding a method to allow the logging of `b256` type in the standard library.

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1083,6 +1083,7 @@ dependencies = [
  "sway-types",
  "sway-utils",
  "term-table",
+ "terminal-link",
```
Comment (adlerjohn) [inline Cargo.lock:1086]: revert changes to lockfile

File: sway-lib-std/src/chain.sw [hunk 2]
```diff
@@ -4,6 +4,12 @@ dep chain/auth;
 use ::panic::panic;
 
 // When generics land, these will be generic.
+pub fn log_b256(value: b256) {
```
Comment (adlerjohn) [inline sway-lib-std/src/chain.sw:7]: This needs a test

**Review by adlerjohn — CHANGES_REQUESTED** [review]

File: sway-lib-std/src/chain.sw (hunk 2 above)
Comment (SilentCicero) [reply sway-lib-std/src/chain.sw:7]: Yeah, will write in a second, just wanted to get this started.

File: Cargo.lock (hunk 1 above)
Comment (SilentCicero) [reply Cargo.lock:1086]: Yep, will revert, sorry, must have snuck in from previous PR.

**Review by nfurfaro — APPROVED** [review]
Nice! LG

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**